Error: File 'ui_image_viewer.ui' is not valid

while executing 'C:\Users\Agustin\AppData\Local\Programs\Python\Python310\Lib\site-packages\PySide6\uic -g python ui_image_viewer.ui'
